test/ctia/entity/web_test.clj

```diff
@@ -158,73 +156,77 @@
 (deftest test-judgement-with-jwt-routes
   (test-for-each-store-with-app
    (fn [app]
-     (helpers/set-capabilities! app "foouser" ["foogroup"] "user" all-capabilities)
-     (helpers/set-capabilities! app "baruser" ["bargroup"] "user" #{})
-     (whoami-helpers/set-whoami-response app "45c1f5e3f05d0" "foouser" "foogroup" "user")
-     (whoami-helpers/set-whoami-response app "2222222222222" "baruser" "bargroup" "user")
-
-     (testing "POST /ctia/judgement"
-       (let [{judgement :parsed-body
-              status :status}
-             (POST app
-                   "ctia/judgement"
-                   :body new-judgement-1
-                   :headers {"Authorization" "45c1f5e3f05d0"
-                             "origin" "http://external.cisco.com"})
-             judgement-id (id/long-id->id (:id judgement))]
-
-         (is (= 201 status))
-
-         (testing "GET /ctia/judgement/:id with bad JWT Authorization header"
-           (let [response
+     (testing "POST /ctia/judgement with JWT Authorization header"
```

Comment:
All these changes are maintly a refactoring.
You can directly jump to: `(testing "Search must properly filter on client-id" ...)`